.github/workflows/build.yml

```diff
@@ -89,6 +85,20 @@ jobs:
           cd docs
           make requirements html SPHINXOPTS="-W"
 
+  test_minimum_versions:
+    name: Test Minimum Versions
+    timeout-minutes: 20
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
+        with:
+          python_version: "3.7"
+      - uses: jupyterlab/maintainer-tools/.github/actions/install-minimums@v1
+      - name: Run the unit tests
+        run: |
+          pytest -vv -W default enterprise_gateway/tests/test_enterprise_gateway.py || pytest -vv -W default --lf
```

Comment:
From the description...
> Add minimum version check job - tests a subset since the 2 failing tests gave no meaningful output

is the inclusion of `enterprise_gateway/tests/test_enterprise_gateway.py` something we should revisit down the road at some point?

Reply:
Potentially yes, I was lazy and didn't want to debug the two failing tests.  :smile:

Reply:
Thanks!